test/test_arhp_wrong_db.rb

```diff
@@ -10,10 +10,10 @@ def setup
     else
       Phenix.load_database_config
     end
-    ActiveRecordHostPool::PoolProxy.class_variable_set(:@@_connection_pools, {})
   end
 
   def teardown
+    ActiveRecordHostPool::PoolProxy.class_variable_set(:@@_connection_pools, {})
     Phenix.burn!
```

Comment:
In most other places we set this class variable, we also call disconnect:
```rb
ActiveRecord::Base.connection.disconnect!
ActiveRecordHostPool::PoolProxy.class_variable_set(:@@_connection_pools, {})
Phenix.burn!
```
I wonder if we (for consistency, if nothing else) should do the same here?

Reply:
So, the reason we don't include it here is because this is testing what happens when you try to connect to a database that doesn't exist. Because it doesn't exist, it never actually establishes a connection, and then attempting to call `disconnect!` raises this error:

`ActiveRecord::ConnectionNotEstablished: No connection pool for 'ActiveRecord::Base' found.`